### ISSUE TITLE
AstroCLIP: a cross-modal foundation model for galaxies

### ISSUE THREAD

**Issue by vishalsudhakar** [vishalsudhakar/thelearningcurve#6]
### Paper Title
AstroCLIP: a cross-modal foundation model for galaxies

### Abstract
We present AstroCLIP, a single, versatile model that can embed both galaxy images and spectra into a shared, physically meaningful latent space. These embeddings can then be used - without any model fine-tuning - for a variety of downstream tasks including (1) accurate in-modality and cross-modality semantic similarity search, (2) photometric redshift estimation, (3) galaxy property estimation from both images and spectra, and (4) morphology classification. Our approach to implementing AstroCLIP consists of two parts. First, we embed galaxy images and spectra separately by pretraining separate transformer-based image and spectrum encoders in self-supervised settings. We then align the encoders using a contrastive loss. We apply our method to spectra from the Dark Energy Spectroscopic Instrument and images from its corresponding Legacy Imaging Survey. Overall, we find remarkable performance on all downstream tasks, even relative to supervised baselines. For example, for a task like photometric redshift prediction, we find similar performance to a specifically-trained ResNet18, and for additional tasks like physical property estimation (stellar mass, age, metallicity, and sSFR), we beat this supervised baseline by 19\% in terms of R2. We also compare our results to a state-of-the-art self-supervised single-modal model for galaxy images, and find that our approach outperforms this benchmark by roughly a factor of two on photometric redshift estimation and physical property prediction in terms of R2, while remaining roughly in-line in terms of morphology classification. Ultimately, our approach represents the first cross-modal self-supervised model for galaxies, and the first self-supervised transformer-based architectures for galaxy images and spectra.

### Link to the Paper
https://doi.org/10.1093/mnras/stae1450

### Why This Paper?
This paper describes an ML technique to combine both photometric and spectroscopic data for predictions of things like redshift and stellar mass.

### Key Topics / Methods
Embedding space, cross-modal contrastive techniques, zero- vs. few-shot predictions

### Suggested by
Brandon Pries
